backend/app/Main.hs

```diff
@@ -4,7 +4,28 @@ module Main
 
 import Universum
 
-import Stub (stub)
+import qualified Data.Yaml as Y
```

Comment:
because it's piece of overcomplicated shit. Do you even know why loot config was created and why do we need to use it at all? (I really doubts)

also I tried to use it with current resolver and there are a lot of compile time errors. feel free to add it yourself :putya: